narwhals/_pandas_like/namespace.py

```diff
@@ -108,6 +108,37 @@ def all(self) -> PandasExpr:
             implementation=self._implementation,
         )
 
+    def lit(self, value: Any, dtype: dtypes.DType | None) -> PandasExpr:
+        def _lit_pandas_series(df: PandasDataFrame) -> PandasSeries:
+            if isinstance(value, (list, tuple)):
+                data = [value for _ in range(len(df._dataframe))]
+                index = df._dataframe.index
+            else:
+                data = [value]
+                index = [0]
```

Comment:
we don't have nested datatypes in Narwhals yet, shall we save on some complexity (until we have a use case) and just do
```
if isinstance(value, (list, tuple)):
    raise NotImplementedError(<some informative error message>)
data = [value]
index = df._dataframe.index[0:1]
```
?

Reply:
Sounds good! Let's keep it simple :)

Out of curiosity, is there a difference between `index = [0]` and `index = df._dataframe.index[0:1]`? Both seems to work fine but I guess I am missing something. When is the `index` used?

Reply:
when aligning different objects (in the `validate_`) functions. the difference is just that if `df` is already length 1, then using `df._dataframe.index[0:1]` won't cause an unnecessary `set_axis` call